server.js

```diff
@@ -4,12 +4,49 @@ const path = require('path');
 
 const app = express();
 
-// view engine setup
 app.set('views', path.join(__dirname, 'views'));
 app.set('view engine', 'hbs');
 app.use(express.static(path.join(__dirname, 'public')));
 
-// add your routes here :)
+app.get('/:code', (req, res) => {
+  const code = req.params.code;
+  const promise1 = fetch('http://localhost:3000/code/' + code)
+  .then(response => response.json());
+  const promise2 = fetch('http://numbersapi.com/' + code)
+  .then(response => response.text());
+
+  Promise.all([promise1, promise2])
+  .then((arr) => {
+    res.render('code', { 
+      'code': arr[0].code, 
```

Comment:
For objects in javascript, quotes around the `key` is not always necessary.

Example
```js
const someKey = 'some';

const someObj = {
    foo: 'bar',
   'foo-bar': 'baz',
 // foo-bar: 'baz',  <--- This is not allowed
    [someKey]: 'qux'
};


```